### ISSUE TITLE
Provide Proxy Support

### ISSUE THREAD

**Issue by buggsdummy** [metacloud/molecule#575]
Please provide proxy support to this, I've tried setting the proxy environment in the playbook, but it doesn't look to be working. Thanks! 

Errors: 
    Could not retrieve mirrorlist http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=os&infra=container error was
12: Timeout on http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=os&infra=container: (28, 'Resolving timed out after 30538 milliseconds')
    The command '/bin/sh -c bash -c 'if [ -x "$(command -v yum)" ]; then yum makecache fast && yum update -y && yum install -y python sudo; fi'' returned a non-zero code: 1
Build failed for molecule_local/centos:latest

$ docker images
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
<none>              <none>              800c0fd93df7        18 hours ago        196.8 MB
ubuntu              latest              f753707788c5        12 days ago         127.2 MB
centos              7                   980e0e4c79ec        7 weeks ago         196.8 MB
centos              latest              980e0e4c79ec        7 weeks ago         196.8 MB
hello-world         latest              c54a2cc56cbb        3 months ago        1.848 kB

**Comment by retr0h** [comment]
@buggsdummy - you want to provide a proxy when building the initial image or a proxy when converging ansible or both?

**Comment by lxhunter** [comment]
Proxy Support for converging would be great! 🍺

**Comment by retr0h** [comment]
> Proxy Support for converging would be great! 🍺

Can you please provide further information on your use case.  Please state where you run molecule from, why you need a proxy, etc..  This will help me understand this feature better.

**Comment by buggsdummy** [comment]
I'm working behind a firewall and I don't see anywhere where I can set a proxy when I'm converging and building my docker instances for testing. It just fails, as I can't get anywhere from yum.

**Comment by retr0h** [comment]
@buggsdummy this feature has been merged.  If you wish to use it you need to deploy off master and update your molecule.yml with the following for Centos.

See discussion on (stackoverflow)[http://stackoverflow.com/questions/22179301/how-do-you-run-apt-get-in-a-dockerfile-behind-a-proxy].

```
docker:
  containers:
    - name: foo
      image: ubuntu
      image_version: latest
      environment:
        http_proxy=ip:3128 
        https_proxy=ip:3128
```

**Comment by retr0h** [comment]
This feature was released with molecule 1.13 which is live on pypi.
